src/Exception/InternalServerErrorHttpException.php

```diff
@@ -4,7 +4,7 @@
 
 use Symfony\Component\HttpKernel\Exception\HttpException;
 
-class UploadHttpException extends HttpException
+class InternalServerErrorHttpException extends HttpException
```

Comment:
Do you mean internal server error as in HTTP 500? Because if you do I don't think is correct, 5XX errors mean there is something wrong with the server, you probably want to throw a 4XX error (client error)

Reply:
I want to throw 5xx as in some cases the php has a [maximum upload size](https://www.php.net/manual/en/ini.core.php#ini.upload-max-filesize). In that case the problem is on the server side.